src/mapd-con-es6.js

```diff
@@ -680,11 +680,11 @@ class MapdCon {
    * Get the privileges for the current user for a specified database object.
    * @param {String} objectName - The name or ID of the object.
    * @param {TDBObjectType} type - The type of the database object.
-   * @return {Promise} Returns the list of database object names (strings).
+   * @return {Promise.<TDBObject[]>} Returns the list of database object names (strings).
```

Comment:
same applies here..  Not a list of strings or "names".. objects.

![image](https://user-images.githubusercontent.com/496983/52244686-5d811900-28a4-11e9-89ee-ac43677c04ed.png)